framework/components/ecl-accordions/_ecl-accordions.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * Accordions
+ * @define accordions
+ */
+
+.ecl-accordions__header {
```

Comment:
should use "accordion" instead of "accordions" for classes, as this is what we do for other components.